code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -518,10 +518,11 @@
 	return null
 
 //Handle a mob struggling
-// Called from /mob/living/carbon/relaymove()
-/obj/belly/proc/relay_resist(var/mob/living/R)
+// Called from /mob/living/proc/process_resist()
+/obj/belly/contents_resist(var/mob/living/R)
```

Comment:
As far as I can tell, the linter is unhappy with this proc, only change is that it's now related to a proc that is annotated to not sleep. It's called from where it should have been (if the logic weren't broken, returning before reaching it), so if it's just an expectation, and not a functional change, it seems to be fine?

Reply:
oh
hm
let me check in a moment

Reply:
> oh hm let me check in a moment

Have you checked?